music21/volume.py

```diff
@@ -254,14 +259,15 @@ def getRealized(
                     # double scalar (so range is between 0 and 1) and scale
                     # the current val (around the base)
                     val = val * (dm.volumeScalar * 2.0)
-            # userArticulations can be a list of 1 or more articulation objects
+            # useArticulations can be a list of 1 or more articulation objects
             # as well as True/False
             if useArticulations is not False:
-                if common.isIterable(useArticulations):
-                    am = useArticulations
-                elif (hasattr(useArticulations, 'classes')
-                       and 'Articulation' in useArticulations.classes):
+                am: Iterable[articulations.Articulation]
+                if isinstance(useArticulations, articulations.Articulation):
                     am = [useArticulations]  # place in a list
+                elif common.isIterable(useArticulations):
+                    assert useArticulations is not True
```

Comment:
remove or disable -- using classes are not responsible for corner cases on main classes.